### ISSUE TITLE
Alternative Docker Registries

### ISSUE THREAD

**Issue by bblommers** [getmoto/moto#6079]
### TLDR:

Our Docker images are now hosted in two locations:
DockerHub: https://hub.docker.com/r/motoserver/moto
GHCR: https://github.com/getmoto/moto/pkgs/container/motoserver

The DockerHub registry may become unavailable, so consider switching to GHCR if possible.

-----------------------------------------

Docker has decided to sunset free Team organizations, such as `motoserver`, and now require us to pay if we want to keep this account and all images within it.
Moto is quite a power-user, with over 1M+ pulls in total, so it is not surprising that Docker wants a reimbursement. But as an Open Source project with no income, this is not an option for us. 

#### Potential lifeline
Docker has a OpenSource program that would allow Moto to continue to use our account, at no cost. We have applied, and have been accepted, into this program. 

However:

1. This only buys us one year, before we have to re-apply. 
2. Docker has long neglected this program, and there is no history of support and commitment
3. We have no information regarding any limits (rate limits, storage size, etc) that apply

So this may well be a stop gap solution that just delays the inevitable.

#### Long-term solution
As stated above, our old releases are now also available on GHCR.
We will start pushing new releases to both registries, to ensure consistency and availability if a registry ever becomes unavailable.

Relevant reading material:

https://news.ycombinator.com/item?id=35154025
https://github.com/docker/hub-feedback/issues/2314

Please let us know if you have any questions or concerns.
